priv/static/js/app.js

```diff
@@ -24,143 +24,155 @@ const handle_scanned_ssid = function (data, table_id) {
-    ssidElem.innerHTML = data.ssid;
+      console.log("data");
+      console.log(ap);
 
-    var securityElem = row.getElementsByClassName("securityCell")[0] || row.insertCell(0);
-    securityElem.setAttribute("class", "securityCell");
+      if (ap.flag && ap.flags.length > 0) {
+        securityElem.innerHTML = `
+        <span>&#128274;</span>
+        `;
+      } else {
+        securityElem.innerHTML = `
+        <span></span>
+        `;
+      }
 
-    if (data.flags.length > 0) {
-      securityElem.innerHTML = `
-      <span>&#128274;</span>
+      var iconsElem = row.getElementsByClassName("iconsCell")[0] || row.insertCell(0);
+      iconsElem.setAttribute("class", "iconsCell");
+      iconsElem.innerHTML = `
+      <svg class="i-sig" x="0px" y="0px" viewBox="0 0 11.26 8">
+      <path class="i-sig-w0" d="M6.337,7.247c-0.391-0.391-1.023-0.391-1.414,0l0.708,0.708L6.337,7.247z"></path>
+      <g class="i-sig-w">
+         <path fill="none" stroke="#000000" stroke-miterlimit="10" d="M7.62,5.966c-1.098-1.098-2.88-1.098-3.977,0"></path>
+         <path class="i-sig-wm" fill="none" stroke="#000000" stroke-miterlimit="10" d="M9.31,4.275C7.278,2.244,3.984,2.245,1.952,4.276"></path>
+         <path class="i-sig-wo" fill="none" stroke="#000000" stroke-miterlimit="10" d="M10.9,2.684c-2.911-2.911-7.629-2.911-10.54,0"></path>
+      </g>
+      </svg>
       `;
-    } else {
-      securityElem.innerHTML = `
-      <span></span>
-      `;
-    }
-
-    var iconsElem = row.getElementsByClassName("iconsCell")[0] || row.insertCell(0);
-    iconsElem.setAttribute("class", "iconsCell");
-    iconsElem.innerHTML = `
-    <svg class="i-sig" x="0px" y="0px" viewBox="0 0 11.26 8">
-    <path class="i-sig-w0" d="M6.337,7.247c-0.391-0.391-1.023-0.391-1.414,0l0.708,0.708L6.337,7.247z"></path>
-    <g class="i-sig-w">
-       <path fill="none" stroke="#000000" stroke-miterlimit="10" d="M7.62,5.966c-1.098-1.098-2.88-1.098-3.977,0"></path>
-       <path class="i-sig-wm" fill="none" stroke="#000000" stroke-miterlimit="10" d="M9.31,4.275C7.278,2.244,3.984,2.245,1.952,4.276"></path>
-       <path class="i-sig-wo" fill="none" stroke="#000000" stroke-miterlimit="10" d="M10.9,2.684c-2.911-2.911-7.629-2.911-10.54,0"></path>
-    </g>
-    </svg>
-    `;
-    iconsElem.children[0].classList.add(signal_to_class(data.signal));
-    return;
-  }
+      iconsElem.children[0].classList.add(signal_to_class(ap.signal_percent));
+      return;
+    })
+  };
 }
 
-const socket = new WebSocket("ws://" + location.host + "/socket");
-socket.onopen = function () {
-  console.log("connected");
-}
+const parseResponse = (response) => response.json();
 
-socket.onmessage = function (event) {
-  var payload = JSON.parse(event.data);
-  switch (payload.type) {
-    case "wifi_scan": {
-      handle_scanned_ssid(payload.data, "wifi_scan");
-      break;
-    }
-    case "wifi_cfg": {
-      handle_scanned_ssid(payload.data, "wifi_cfg");
-      break;
+fetch("/api/v1/access_points")
```

Comment:
I'm not seeing where the `access_points` get polled for updates. Did I miss it or is it coming?

Reply:
Forgot that piece I will do that.